MapboxNavigation/NavigationMapView.swift

```diff
@@ -1370,9 +1372,22 @@ extension NavigationMapView {
         style?.removeLayer(highlightedBuildingsLayer)
     }
     
+    private func addBuildingsSource() -> MGLSource? {
+        let buildingsSource = style?.source(withIdentifier: SourceIdentifier.buildingExtrusion)
+        if buildingsSource == nil {
+            let buildingsSource = MGLVectorTileSource(identifier: SourceIdentifier.buildingExtrusion,
+                                                      configurationURL: URL(string: "mapbox://mapbox.mapbox-streets-v8")!)
```

Comment:
I just realized this code runs even if the style already has a Streets source v8 source, either as part of the `composite` source or independently. This is potentially problematic: the style specification requires that the overall style contain no conflicts between the source layer identifiers of the various sources. By adding a redundant copy of the Streets source, we guarantee conflicts between those copies. In the best-case scenario, this will only result in console spew, but it could also result in runtime styling changes not taking effect uniformly.

Reply:
I missed that this code only runs if `streetsSources.isEmpty` is true. Sorry for the false alarm.